packages/shared/useCounter/index.ts

```diff
@@ -1,16 +1,38 @@
 import { ref } from 'vue-demi'
 
+interface useCounterOptions {
+  minValue?: number
+  maxValue?: number
```

Comment:
```suggestion
  min?: number
  max?: number
```

Reply:
@antfu i push a new commit with `min` and `max` instead of `minValue` and `maxValue`